core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/showMessageUtils.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Utility function for logging the progress of the tests.
+ */
+
+let showMessage = async function(message) {
+  await console.log(message);
+};
+
+module.exports = { showMessage };
+
+/** We need to ignore the eslint checks for the console statements in our
```

Comment:
What is this comment for? I don't see anything being ignored here.

Reply:
@seanlip umm yes.. I have left it for now.. for ignoring this console log statement in the eslint checks, I have used 2 ways, one was to add this file in the .eslintignore, and the other was just to simply add a pragma to ignore this line from the console eslint check that was using this pragma: ` // eslint-disable-line no-console`, but the problem was still the lints checks were failing in both the ways.. and surprisingly there was an empty error log where the lints checks were failing but still showing the error that lint checks are failing.

Reply:
OK -- for reference, the pragma would be better. If this is blocking you and is an error in develop, then you can also start a debugging doc and talk to the dev workflow team for advice, but make sure to read the wiki pages on eslint checks first.